examples/ffi-example/src/dictu-include.h

```diff
@@ -0,0 +1,643 @@
+#ifndef dictumod_include_h
```

Comment:
It's probably worth adding this to the include directory within src, that way people may find it easier rather than it being in the examples directory